src/SignalR/clients/ts/signalr-protocol-msgpack/signalr-protocol-msgpack.npmproj

```diff
@@ -12,5 +12,9 @@
     <ProjectReference Include="..\signalr\signalr.npmproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ReplacePackageContent Include="file:../signalr" ReplaceWith="$(PackageVersion)" />
```

Comment:
We should use an npm version range here (see [npm-semver for syntax](https://docs.npmjs.com/misc/semver)). If we want the same behavior we have in NuGet, we need to explicitly specify `>=`. Alternatively we could use `~` which only allows the patch version to float. I think for now, let's match our NuGet behavior:

```suggestion
    <ReplacePackageContent Include="file:../signalr" ReplaceWith=">=$(PackageVersion)" />
```